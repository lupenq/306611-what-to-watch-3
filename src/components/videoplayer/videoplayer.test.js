import Videoplayer from './videoplayer';

const movie = {
  preview: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
};

it(`Render Videoplayer`, () => {
  const tree = renderer
  .create(
      <Videoplayer preview={movie.preview}/>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
