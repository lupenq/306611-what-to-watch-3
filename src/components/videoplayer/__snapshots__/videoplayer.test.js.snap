// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Videoplayer 1`] = `
<video
  autoPlay={true}
  muted={true}
  width="100%"
>
  <source
    src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
  />
</video>
`;
